### DISCUSSION TITLE
piper-whistle

### DISCUSSION THREAD

**Discussion by BlurryRoots** [rhasspy/piper#283] [Show and tell]
Hey everybody, after consolidating a couple of scripts used to help me organize piper voices, this tool was the result. It let's you query available voices and install them from huggingface. Maybe somebody can make use of this as well. You can find the source here on [github](https://github.com/think-biq/piper-whistle) or [gitlab](https://gitlab.com/think-biq/piper-whistle).

**Comment by BlurryRoots** [comment]
Update: piper-whistle is available through pip now `pip install -U piper-whistle`

**Comment by akhil3417** [comment]
thanks, is their any api for piper?

**Reply by BlurryRoots** [reply]
Hey @akhil3417, what sort of API are you thinking of? There are some ways of embedding or scripting. You may want to checkout the [official python binding](https://github.com/rhasspy/piper/tree/master/src/python_run) for some inspiration.
